src/Search/ReturnLastRequestedUri.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Search;
+
+use Psr\Http\Message\UriInterface;
+
+interface ReturnLastRequestedUri
```

Comment:
I don't think this is needed, see https://github.com/cultuurnet/udb3-backend/pull/1556/files#r1455437664